staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/sample_and_watermark_test.go

```diff
@@ -95,9 +95,9 @@ func TestSampler(t *testing.T) {
 }
 
 /* getHistogramCount returns the count of the named histogram */
-func getHistogramCount(regs Registerables, metricName string) (int64, error) {
+func getHistogramCount(registry compbasemetrics.KubeRegistry, metricName string, allowNotFound bool) (int64, error) {
```

Comment:
I'd actually change this so that allowNotFound is not a parameter on the function and return `0, nil` as default fallback behavior. My reasons:

1. cleaner function signature
2. this is a private function which seems to be used exactly once in this file
3. this is quite possibly just actually the correct behavior.

Reply:
The force-push to 9ec70d16b11 simplified that function without losing that bit of information if the test ever does fail that way.